R/functions_prep_project.R

```diff
@@ -35,6 +35,9 @@ apply_sector_split_to_loans <- function(data,
                                         abcd,
                                         companies_sector_split,
                                         sector_split_type) {
+  unique_companies_pre_split <- data %>%
+    distinct(name_abcd)
```

Comment:
Is there a reason to use `name` instead of `id`?

Reply:
yes, data is the `matched_prioritized` loan book, which does not have `company_id`. I don't love this either, but it seemed the logical thing to do